rllib/examples/policy/cliff_walking_wall_policy.py

```diff
@@ -0,0 +1,85 @@
+from ray.rllib.policy.policy import Policy, ViewRequirement
+from ray.rllib.policy.sample_batch import SampleBatch
+from ray.rllib.utils.typing import AlgorithmConfigDict, TensorStructType, TensorType
+from typing import Dict, Union, List, Tuple, Optional
+from ray.rllib.utils.annotations import override
+from ray.rllib.models.torch.torch_action_dist import TorchCategorical
+import numpy as np
+import gym
+
+
+class CliffWalkingPolicy(Policy):
+    """Optimal RLlib policy for the CliffWalking environment with
+    epsilon-greedy exploration"""
```

Comment:
Specify what epsilon greedy means in this context: with prob epsilon you act greedily according to an optimal policy and with 1-eps you act uniformly random across all possible actions.

Reply:
is eps the prob of acting greedily or randomly?

Reply:
Eps is the probability of acting randomly, so epsilon=0.0 is optimal. Will clarify the docstring.